src/components/molecules/AppNavBar/AppNavBar.tsx

```diff
@@ -112,7 +112,21 @@ export default function AppNavBar({ authUser }: Proptypes) {
           <div className={classes.grow} />
           {authUser && (
             <Fragment>
-              <div className={classes.sectionDesktop}>Tracks</div>
+              <div className={classes.sectionDesktop}>
```

Comment:
It is a bit better to pass "Link" from reach router as component prop and add a "to" prop instead of one-click.

It is not very important through

Reply:
sounds good I'll try this